packages/wonder-blocks-link/src/components/__tests__/link.test.js

```diff
@@ -41,16 +35,16 @@ describe("Link", () => {
             );
 
             // Act
-            const linkWrapper = wrapper.find("Link").first();
-            linkWrapper.simulate("click", {button: 0});
+            const linkWrapper = screen.getByText("Click me!");
```

Comment:
Nit: I think this can be just `link` instead of `linkWrapper` now.

Reply:
ah yeah, good call! will update after fixing the failed test.